Source/Scene/Cesium3DTile.js

```diff
@@ -607,6 +609,63 @@ define([
 
     var scratchJulianDate = new JulianDate();
 
+    /**
+     * Get the tile's screen space error.
+     *
+     * @private
+     */
+    Cesium3DTile.prototype.getScreenSpaceError = function(frameState, useParentGeometricError) {
```

Comment:
Is `useParentGeometricError` ever `false`? Is that in a future PR?

Reply:
It's undefined when called from `updateVisibility`. I forgot to change that to false which should make it more explicit.